script/Connector-XCM.s.sol

```diff
@@ -17,8 +17,11 @@ contract ConnectorXCMScript is Script {
         address memberlistFactory_ = address(new MemberlistFactory());
         CentrifugeConnector connector = new CentrifugeConnector(tokenFactory_, memberlistFactory_);
 
-        ConnectorXCMRouter router =
-            new ConnectorXCMRouter(address(connector), address(vm.envAddress("CENTRIFUGE_CHAIN_ORIGIN")));
+        ConnectorXCMRouter router = new ConnectorXCMRouter(
+                address(connector),
+                address(vm.envAddress("CENTRIFUGE_CHAIN_ORIGIN")),
+                vm.envBytes("CENTRIFUGE_CHAIN_HANDLE_CALL_INDEX")
+        );
```

Comment:
What is `vm` and what does `startBroadcast()` do?

Reply:
Here's some context on this, it's related to using Foundry's scripting feature: https://book.getfoundry.sh/tutorials/solidity-scripting

Reply:
The docs say:

```
// Using the address that calls the test contract, has all subsequent calls (at this call depth only) create transactions that can later be signed and sent onchain
```